webapp/src/api/api.ts

```diff
@@ -0,0 +1,49 @@
+import {PlaceType} from "../types/PlaceType";
+
+export async function getPlaces() : Promise<PlaceType[]> {
+    const apiEndPoint = "http://localhost:5000/api"
+    let response = await fetch(apiEndPoint + "/places/list");
+    return response.json();
+}
+
+export async function addPlace(place: PlaceType) : Promise<boolean> {
+    const apiEndPoint = "http://localhost:5000/api"
+    let response = await fetch(apiEndPoint + "/places/add", {
+        method: "POST",
+        headers: {"Content-Type": "application/json"},
+        body: JSON.stringify({
+            "title": place.title,
+            "uuid": place.uuid,
+            "longitude": place.longitude,
+            "latitude": place.latitude
+        })
+    });
+    return response.status === 200;
+}
+
+export async function deletePlace(title: string) : Promise<boolean> {
+    const apiEndPoint = "http://localhost:5000/api"
```

Comment:
We have to take into account that may be in the future this url must be changed as the Web app would be deployed in a machine